exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -31,10 +31,10 @@ Ad the following code to the top of your solution file to use the `date_time` pa
 :- use_module(library(date_time)).
 ```
 
-```exercism/note
-Tip: the [source of the date_time package][https://www.swi-prolog.org/pack/file_details/date_time/prolog/date_time.pl?show=src] is well documented and be a great help!
-```
+### Exercism note
+Tip: the [source of the date_time package][date-time-package-source] is well documented and be a great help!
+
```

Comment:
[We do have such a non-standard block (admonition)](https://exercism.org/docs/building/markdown/markdown#h-special-blocks-sometimes-called-admonitions), but it is marked up differently now. And it requires to have the link definition inside the block, after a separating line:

```suggestion
~~~~exercism/note
Tip: the [source of the date_time package][date-time-package-source] is well documented and be a great help!

[date-time-package-source]: https://www.swi-prolog.org/pack/file_details/date_time/prolog/date_time.pl?show=src
~~~~
```